cachito/nexus_scripts/configure_nexus.groovy

```diff
@@ -0,0 +1,139 @@
+// This script configures Nexus to be ready for Cachito and is idempotent
```

Comment:
I was thinking it would expand to cover other languages. In the future, we could split up in the following way:
* General configuration
* JavaScript repositories
* Other language repositories